scripts/vendor

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+set -eu
+
+TYP=$1
+
+usage() {
+  echo "usage: ./scripts/vendor <update|outdated>"
+  exit 1
+}
+
+if [ -z "$TYP" ]; then
+  usage
+fi
+
+# create dummy go.mod, see comment in vendor.mod
+cat > go.mod <<EOL
+module github.com/docker/cli
+
+go 1.16
+EOL
+
+update() {
+  go mod tidy -modfile=vendor.mod
+  go mod vendor -modfile=vendor.mod
+
+  if [ ! -x "$(command -v modvendor)" ]; then
+    echo "modvendor not found. Install with 'go install github.com/goware/modvendor@v0.5.0'"
+    exit 1
+  fi
```

Comment:
I'd recommend doing something similar to https://github.com/docker/swarmkit/pull/3046/commits/a46370c9638f3f19bb08ede2a6fe871985486bbb, that way tools versions can be tracked in a separate `go.mod` file and will not interfere with actual runtime dependencies.

Reply:
Yep sounds better with go build tools, will make some changes.

Reply:
Hum will do that in a follow-up actually, there are other tools that would need the same behavior like `goversioninfo`.